src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Models/Apprentice/InformRequest.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using SFA.DAS.Authorization.ModelBinding;
+
+namespace SFA.DAS.ProviderCommitments.Web.Models.Apprentice
+{
+    public class InformRequest : DetailsRequest
```

Comment:
I think it would be better to not inherit, and instead add the properties we need explicitly into the InformRequest, in order to keep things separate.